Source/Urho3D/Graphics/RibbonTrail.cpp

```diff
@@ -53,13 +53,31 @@ inline bool CompareTails(TrailPoint* lhs, TrailPoint* rhs)
     return lhs->sortDistance_ > rhs->sortDistance_;
 }
 
+TrailPoint::TrailPoint() :
+    TrailPoint{Vector3::ZERO, Vector3::ZERO}
+{
+}
+
+TrailPoint::TrailPoint(const Vector3& position, const Vector3& forward) :
+    position_{position},
+    forward_{forward},
+    parentPos_{Vector3::ZERO},
+    elapsedLength_{0.0f},
```

Comment:
I suggest to move scalar initializers into header, mark `TrailPoint()` as `=default`, and keep only `TrailPoint(const Vector3& position, const Vector3& forward) : position_(...), forward_(...) {}`

Reply:
Ok sure, will do that.